templates/index.html

```diff
@@ -37,15 +38,13 @@
             <label class="heading" for="search_engine">Default Search Engine</label>
             <div class="input-group margin-bottom-sm">
               <span class="input-group-addon"><i class="fa fa-search fa-fw"></i></span>
-              <input pattern="https?://.+" class="form-control" placeholder="Enter link" name="search_engine" title="only url is allowed"
+              <input pattern="" class="form-control" placeholder="Enter link" name="search_engine" title="only url is allowed"
                  type="url"></div>
             <p id="scroll-to">Default search engine of the browser</p>
           </div>
           <br>
           <div class="form-group">
-
-            <label class="heading" for="bookmark">Choose Either ON or OFF:<span>
-                <p id="scroll-to">Packages that will be installed</p>
+            <label class="heading" for="bookmark">Packages that are checked will be installed<span>
```

Comment:
Please don't change the intendation

Reply:
Please do changes as recommended by @tabesin previously